### PR TITLE
Added allowInputToggle option

### PR DESCRIPTION
**Pull request by jevanlingen**
If `true`, the picker will show on textbox focus and icon click.

### CODE REVIEW

File: addon/components/bs-datetimepicker.js [hunk 1]
```diff
@@ -65,7 +65,8 @@ export default Component.extend(DynamicAttributeBindings, {
       useCurrent: this.getWithDefault('useCurrent', false),
       viewDate: this.getWithDefault('viewDate', defaults.viewDate),
       viewMode: this.getWithDefault('viewMode', defaults.viewMode),
-      widgetPositioning: this.getWithDefault('widgetPositioning', defaults.widgetPositioning)
+      widgetPositioning: this.getWithDefault('widgetPositioning', defaults.widgetPositioning),
+      allowInputToggle: this.getWithDefault('allowInputToggle', defaults.allowInputToggle)
```
Comment (btecu) [inline addon/components/bs-datetimepicker.js]: Do you mind keeping the options alphabetically ordered?

**Comment by btecu** [comment]
Needs to be documented.

**Comment by jevanlingen** [comment]
@btecu: Could you release a new version with the latest options included?

**Comment by btecu** [comment]
Released as `0.6.1`.

**Pull request merged**